### PR TITLE
Found From Googleing more

### PR DESCRIPTION
**Pull request by nhyty**
All web links used on the people pages

### CODE REVIEW

**Comment by nhyty** [comment]
Anyone fancy reviewing

File: _people/christey_nethercott.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Christey Nethercott
+headshot: dgWk7Dw
+gender: female
+submitted: false
+
+links:
+  - type: starnow
```
Comment (samozzy) [inline _people/christey_nethercott.md]: Do we support this yet? [I can't see it](https://github.com/newtheatre/history-project/blob/master/_data/link-types.yaml). Do all different types need to be defined in that yaml doc?

File: _people/douggie_mcmeekin.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Douggie McMeekin
+headshot: TXqLQv2
+gender: male
+submitted: false
+course:
+  - Physics
+
+links:
+  - type: The Artist Partnership
```
Comment (samozzy) [inline _people/douggie_mcmeekin.md]: See above.

**Comment by wjdp** [comment]
Yeah, that was the fix I did yesterday. They just show as generic links

File: _people/christey_nethercott.md (hunk 1 above)
Comment (wjdp) [inline _people/christey_nethercott.md]: Can this be changed to `StarNow` as used to make the text for the link.

**Pull request merged**